testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -552,6 +556,12 @@
 LAMMPSDUMP_chain2 = (_data_ref / "lammps/chain_dump_2.lammpstrj").as_posix()
 LAMMPS_chain = (_data_ref / "lammps/chain_initial.data").as_posix()
 LAMMPSdata_many_bonds = (_data_ref / "lammps/a_lot_of_bond_types.data").as_posix()
+LAMMPSdata_additional_columns = (
```

Comment:
Can you make one line like the rest please?

Reply:
Does PEP8 not apply to the testsuite?

Reply:
It does in general, just not to this file :smiley:, sorry for any confusion. I think we just decided that one per line was easiest for readability. Sorry for confusion, I know this is inconsistent. :+1:

Reply:
No worries, makes total sense! pep8speaks was the one I felt pressured by 😁